source/content/guides/integrated-composer/02-dependencies.md

```diff
@@ -5,14 +5,29 @@ description: Learn how to add or remove an individual site dependency.
 tags: [composer, workflow]
 categories: [get-started]
 contributors: [ari, edwardangert]
-reviewed: "2022-04-28"
 layout: guide
 showtoc: true
 permalink: docs/guides/integrated-composer/dependencies
 anchorid: dependencies
 ---
 
-This sections provides information on how to add or remove individual site dependencies.
+This section provides information on dependency requirements and how to add or remove individual site dependencies.
+
+## Composer `require` and `require-dev` Sections
+
+It's important to understand how `require` and `require-dev` are used on the Pantheon platform.
+
+### Composer `require` Section
+
+You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. Drupal modules / themes and WordPress plugins / themes should always be in the `require` section, not the `require-dev` section. All dependencies in the `require` section are pushed to Pantheon Dev and Multidev environments. 
+
+You can use  `--no-dev` to prevent specific dependencies from being pushed to your Dev and Multidev environments.
```

Comment:
Sorry to insist on this:

```
You really don't have control over this. It will always be like this:

- dev dependencies pushed to dev/multidev
- dev dependencies NOT pushed to test/live
```

You can control what goes to "require" vs "require-dev" as explained in this page but Pantheon will always install dev dependencies to dev/multidev and will always NOT install dev dependencies to test/live (btw, maybe this is worth noticing it in this doc)

Reply:
@kporras07 Can the -- no dev  option that prevents dev dependencies from being installed be used in both the require and require dev sections?

Reply:
"--no-dev" is an option you could pass if you are running `composer install` in your local machine. In Pantheon you can not control that.

The impact of that option is to not install the dev dependencies. So that option is always used in test/live environments and never used in dev/multidev but the user has no option to control that.

Happy to chat more about this if it's unclear :)